eng/common/Extract-ReleaseNotes.ps1

```diff
@@ -1,7 +1,8 @@
 # given a CHANGELOG.md file, extract the relevant info we need to decorate a release
 param (
   [Parameter(Mandatory = $true)]
-  [String]$ChangeLogLocation
+  [String]$ChangeLogLocation,
```

Comment:
While you can test changes to the script here, ultimately any changes will need to be made in the azure-sdk-tools repo.